float64 x
float64 y
int64 radius
---
int64 A